docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -10,113 +10,155 @@ You can push {es-sec} cases to these third-party systems:
 * {ibm-r}
 * {swimlane}
 
+IMPORTANT: To create connectors and send cases to external systems, you need the
+https://www.elastic.co/subscriptions[appropriate license], and your role needs *All* privileges for the *Action and Connectors* feature. For more information, refer to <<case-permissions>>.
+
 To push cases, you need to create a connector, which stores the information
 required to interact with an external system.
 
 After you have created a connector, you can set {es-sec} cases to
 automatically close when they are sent to external systems.
 
-NOTE: To create connectors and send cases to external systems, you need the
-https://www.elastic.co/subscriptions[appropriate license] and your role needs *All* privileges for the *Action and Connectors* feature. For more information, see <<case-permissions>>.
-
-[[create-new-connector]]
 [float]
+[[create-new-connector]]
 === Create a new connector
 
 . Go to *Investigate* -> *Cases* -> *Edit external connection*.
 +
 [role="screenshot"]
 image::images/cases-ui-connector.png[Shows the page for creating connectors]
 . From the *Incident management system* list, select *Add new connector*.
-. Select one of these:
-* *{sn}*: To send cases to {sn}
+. Select the system to send cases to: *{sn}*, *{jira}*, *{ibm-r}*, or *{swimlane}*.
```

Comment:
There is setup that's required on the Servicenow instance before it can be used. Should those be included here? It's in the stack connector docs: https://www.elastic.co/guide/en/kibana/master/servicenow-action-type.html

Reply:
Are you referring to users needing to install the [Elastic for ITSM](https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88) and the [Elastic for Security Operations (SecOps)](https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f) applications from the ServiceNow Store before they can use the connectors?

Reply:
No, for OAuth, they also need to [create an RSA keypair](https://www.elastic.co/guide/en/kibana/master/servicenow-action-type.html#_create_an_rsa_keypair_and_add_an_x_509_certificate) and [create an OAuth JWT API endpoint](https://www.elastic.co/guide/en/kibana/master/servicenow-action-type.html#_create_an_oauth_jwt_api_endpoint_for_external_clients_with_a_jwt_verifiers_map)

Reply:
Ah, gotcha. So I loosely refer to those sections in the description for the Use OAuth authentication toggle, but I wasn't able to link to either sections directly because they don't have anchor text at the top. I'll need to open a PR against those 3 topics in the Kibana repo to add the anchor text. 
cc: @lcawl